modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,61 @@
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
+= Widening the mirror registry scope of the ImageContentSourcePolicy object
+
+Use the `--icsp-scope=registry` flag with the `oc adm catalog mirror` command to widen the mirror registry scope in the `ImageContentSourcePolicy` object. This minimizes the number of times the cluster nodes are rebooted when a registry entry is updated.
+
+The `oc adm catalog mirror` command generates an `ImageContentSourcePolicy` resource that maps the original container image repository to a new location where it is mirrored, typically inside a disconnected environment. When a new or modified `ImageContentSourcePolicy` resource is applied to a cluster, it is converted into a configuration file for CRI-O and placed onto each node. When you update the configuration file for a node, the node reboots. A widely scoped `ImageContentSourcePolicy` resource reduces the number of times the resource needs to change and thus reduces the number of times cluster nodes have to reboot.
```

Comment:
Passive voice. Perhaps "The `oc adm catalog mirror` command generates an `ImageContentSourcePolicy` resource that maps the original container image repository to a new location and mirrors it. This typically occurs inside a disconnected environment."

Reply:
What converts it? Passive voice typically happens when we don't know the subject of the action. Is it OpenShift generally, the `oc adm catalog mirror` command, the apply command, or something else? What is CRI-O?